source/conf.py

```diff
@@ -42,8 +42,18 @@
     'sphinxext.linkcheckdiff',
     'hoverxref.extension',
     'notfound.extension',
+    'versionwarning.extension',
 ]
 
+versionwarning_messages = {
+    "latest": "The documentation you are currently viewing is for upcoming changes to the WPILib library." +
```

Comment:
Would "...is for upcoming changes to WPILib" be enough? Adding "library" to the end of "WPILib" seems a little redundant.